src/olympia/addons/templates/addons/impala/button.html

```diff
@@ -48,7 +50,16 @@
       {{ _('End-User License Agreement') }}
     </a>
   {% endif %}
-
+  {% if waffle.switch('webext-permissions') %}
+    <br>
+    <a class="webext-permissions badge" href="{{ url('addons.privacy', addon.slug) }}">
```

Comment:
I suspect you want `href="#"` here or link to a real page that has the permission list, and not the privacy view.